src/py_semantic_taxonomy/adapters/routers/router.py

```diff
@@ -12,19 +12,24 @@
 from py_semantic_taxonomy.dependencies import get_graph_service, get_search_service
 from py_semantic_taxonomy.domain import entities as de
 
-router = APIRouter()
+router = APIRouter(prefix="/v1")
 
 
 class Paths(StrEnum):
-    concept = "/concept/"
-    concept_scheme = "/concept_scheme/"
-    catchall = "/{_:path}"
-    relationship = "/relationships/"
-    correspondence = "/correspondence/"
-    association = "/association/"
-    made_of = "/made_of/"
-    search = "/concept/search/"
-    suggest = "/concept/suggest/"
+    concepts = "/concepts"
+    concept = "/concepts/{iri:path}"
+    concept_schemes = "/concept_schemes"
+    concept_scheme = "/concept_schemes/{iri:path}"
+    catchall = "{_:path}"
```

Comment:
The reason the catchall is there (as a last resort) is so that when we deply this at `vocab.whatever`, our URIs are actually available, i.e. `https://vocab.whatever/{concept_scheme}/{concept}` can be copy-pasted into the web browser, and our server can *also* live at `https://vocab.whatever/`. This might seem like a minor point but I promise it is a big deal, especially for non-technically oriented users.

I don't know how to do this with e.g. NGINX rewrite rules as we don't know the format in advance (people don't have to follow this pattern). Instead, we can just check if we have the URI in our system, and redirect as appropriate.

Reply:
I know this is a big deal, but as it was it didn't work. We have some options here:
- make sure routes are registered in order so the catchall doesn't catch higher priority routes,
- make a separate enough route pattern 
- separate web application from the web semantic API, and use a redirect based on [accepted content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Accept) (e.g. `jsonld` vs `text/html`)